### DISCUSSION TITLE
Add building=maisonette tag

### DISCUSSION THREAD

**Discussion by twomagi** [streetcomplete/StreetComplete#4638] [Ideas]
As a regular user of Street Complete please can I ask that you consider adding building=maisonette tag for use in the UK.
These are building where each flat has a separate door to the outside from other flats in the same building.
Thank you

**Comment by HolgerJeromin** [comment]
This tag has only 670 uses world wide and no wiki documentation page
https://taginfo.openstreetmap.org/tags/building=maisonette#overview

This is far too few and unspecified to include in this app.
